radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<h3>Pros</h3>
+<ul>
+<li></li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li></li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+SLO Computer,Incident Response,Hold,none,"<p><a href=""https://github.com/last9/slo-computer"">SLO Computer</a> is building a sandbox for our DevOps and SRE community - SLO computer - a product that makes setting and monitoring SLOs for all your services intuitively seamless and blazingly fast.</p>
+<p>“SLOs, Error windows and alerts are complicated. Here’s an attempt to make it easy.”</p>
+<h3>Pros</h3>
+<ul>
+<li></li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li></li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Rootly,Incident Response,Hold,none,"<p><a href=""https://rootly.com"">Rootly</a> is an incident management platform and Slackbot designed to help companies resolve incidents faster by automating manual admin tasks and providing insight to prevent them in the future</p>
+<h3>Pros</h3>
+<ul>
+<li>Manage incidents directly from Slack</li>
+<li>Say goodbye to manual copy-paste timelines</li>
+<li>Encode entire incident process.</li>
+<li>Integrate with favorite tools</li>
+<li>Reduce context switching</li>
+<li>Faster retrospectives</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li></li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Grafana OnCall,Incident Response,Hold,none,"<p><a href=""https://grafana.com/oss/oncall/"">Grafana OnCall</a> is an open source, easy-to-use on-call management tool built to help teams improve their collaboration and resolve incidents faster.</p>
+<p>Grafana OnCall (formerly known as Amixr) was started in 2018 at Amixr Inc., which Grafana Labs acquired in 2021. Launched on Grafana Cloud in late 2021 and open sourced in 2022 for users needing self-managed and on-premises deployments, Grafana OnCall provides an on-call management solution that is simple to use, accessible to everyone, and allows engineers to actually sleep well at night knowing that only the relevant notifications will reach them. Grafana Labs uses Grafana OnCall to run its own cloud services.</p>
+<h3>Pros</h3>
+<ul>
+<li>On-call management</li>
+<li>Detection</li>
+<li>Alert grouping and routing</li>
+<li>Notification and collaboration</li>
+<li>Remediation</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li></li>
+</ul>
+<p>When decision made: 5/1/2021</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Keep,Incident Response,Hold,none,"<p><a href=""https://github.com/keephq/keep"">Keep</a> a simple CLI tool that contains everything you need to start creating Alerts.</p>
```

Comment:
There are an awful lot of the "Incident Response" tools marked as `Hold`.  It seems likely that at least some of them should be downgraded to `Divest`.

Reply:
Switched most of them to divest.